spec/models/custom_attribute_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe CustomAttribute do
+  it "return correct string field type" do
+    string_custom_attribute = FactoryGirl.build(:custom_attribute, :name => "foo", :value => "string", :resource_type => 'ExtManagementSystem')
```

Comment:
@alongoldboim  can be FactoryGirls moved into let's (placed under describe) ?
let(:string_custom_attribute)  { FactoryGirl.build(:custom_attribute, :name => "foo", :value => "string", :resource_type => 'ExtManagementSystem') }